reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -334,7 +334,7 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
 
     private fun addRuleViolation(ruleViolation: RuleViolation) {
         val resolutions = addResolutions(ruleViolation)
-        val pkg = packages.getValue(ruleViolation.pkg)
+        val pkg = packages[ruleViolation.pkg] ?: createEmptyPackage(ruleViolation.pkg)
```

Comment:
commit-message: 

"mode mapper" -> "model mapper"
"The WebApp reporter so far failed to be generated" -> "The WebApp report so far failed to be generated"
"to by a valid" -> "to be a valid"
"to display such "global" rule violations in the WebApp reporter." -> "to display such "global" rule violations in the WebApp report".